modules/claims_api/db/migrate/20190904156435_create_power_of_attorney.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class CreatePowerOfAttorney < ActiveRecord::Migration[5.2]
+  def change
+    enable_extension 'uuid-ossp'
+    enable_extension 'pgcrypto'
```

Comment:
We do a uuid as a table id instead of integer and pgcrypto seems to be a requirement for doing uuid id's in Rails 5, I can test to see if I can run without calling this in the migration.